functions/spanner/src/main/java/functions/HelloSpanner.java

```diff
@@ -97,27 +97,26 @@ DatabaseClient getClient() throws Throwable {
 
   @Override
   public void service(HttpRequest request, HttpResponse response) throws Exception {
-    BufferedWriter writer = response.getWriter();
+    var writer = new PrintWriter(response.getWriter());
     try {
       DatabaseClient client = getClient();
       try (ResultSet rs =
           client
               .singleUse()
               .executeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"))) {
-        writer.write("Albums:\n");
+        writer.printf("Albums:\n");
         while (rs.next()) {
-          writer.write(
-              String.format(
-                  "%d %d %s\n",
-                  rs.getLong("SingerId"), rs.getLong("AlbumId"), rs.getString("AlbumTitle")));
+          writer.printf(
+              "%d %d %s\n",
```

Comment:
Nit: `spotbugs` does not like `\n`, and prefers `%n` instead.

Apply that comment throughout this file.

Reply:
Will do, though I'll note that I didn't change these strings, which already had `\n` in them.